src/LightSpeed/toolbox.js

```diff
@@ -52,6 +52,18 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
             this._updateBeamTopic);
         this.hideTopic('updateBeam');
 
+        this._spawnPowerupTopic = new LSUserFunction('spawnPowerup');
+        this._spawnPowerupTopic.bindGlobal(this._global);
+        this.addTopic('spawnPowerup', 'NAME', 'go-home-symbolic',
+            this._spawnPowerupTopic);
+        // this.hideTopic('spawnPowerup');
```

Comment:
Why is this line commented? It's for testing purpose?

Reply:
Oh yeah, this should be uncommented now, glad you spotted it.